pgspecial/dbcommands.py

```diff
@@ -1310,3 +1309,36 @@ def sql_name_pattern(pattern):
         schema = '^(' + schema + ')$'
 
     return schema, relname
+
+
+@special_command('\\sf', '\\sf[+] FUNCNAME', 'Show a function\'s definition.', hidden=True)
+def show_function_definition(cur, pattern, verbose):
+    if '(' in pattern:
+        sql = cur.mogrify("SELECT %s::pg_catalog.regprocedure::pg_catalog.oid", [pattern])
+    else:
+        sql = cur.mogrify("SELECT %s::pg_catalog.regproc::pg_catalog.oid", [pattern])
+    log.debug(sql)
+    cur.execute(sql)
+    (foid,) = cur.fetchone()
+
+    sql = cur.mogrify("SELECT pg_catalog.pg_get_functiondef(%s) as source", [foid])
+    log.debug(sql)
+    cur.execute(sql)
+    statusmessage = cur.statusmessage
+    if cur.description:
+        headers = [x[0] for x in cur.description]
+        if verbose:
+            (source,) = cur.fetchone()
+            rows = []
+            rown = None
+            for row in source.splitlines():
+                if rown is None:
+                    if row.startswith('AS '):
+                        rown = 1
+                else:
+                    rown += 1
+                rows.append('%-7s %s' % ('' if rown is None else rown, row))
+            cur = [('\n'.join(rows) + '\n',)]
+    else:
+        headers = None
+    return [(None, cur, headers, statusmessage)]
```

Comment:
Yes, I see, returning `None` as status does the right thing in pgcli, so I'm gonna removing it as well.